README.md

```diff
@@ -121,7 +121,17 @@ If you're using macOS, you can install Cog using Homebrew:
 brew install cog
 ```
 
-You can also download and install the latest release of Cog
+You can run an install script which will download and install the latest release of Cog:
+```
+# via curl
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
+# or via wget:
+sh -c "$(wget -qO- https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
+# or via fetch:
+sh -c "$(fetch -o - https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh)"
```

Comment:
```suggestion
curl -fsSL https://cog.run/installer | sh
# or via wget:
wget -qO- https://cog.run/installer | sh
# or via fetch:
fetch -o - https://cog.run/installer | sh
```

@zeke I don't know where to make a PR to update cog.run, do you know? This does look nicer, though.

Reply:
@chigozienri I think we'd need to set up a custom route handler for that in CloudFlare

Reply:
I set that up:

```
$ curl -I "https://cog.run/installer"
HTTP/2 301 
date: Tue, 19 Mar 2024 02:49:41 GMT
location: https://raw.githubusercontent.com/replicate/cog/main/tools/install.sh
```